src/tests/ftest/container/create.py

```diff
@@ -31,6 +31,7 @@
 
 from daos_api import DaosPool, DaosContainer, DaosApiError
 
+# pylint: disable = redefined-variable-type
```

Comment:
The [line](https://github.com/daos-stack/daos/blob/8be5871e9a0329f841d9f59ed0d25df4b3b2cada/src/tests/ftest/container/open.py#L134) of code that is causing this linting error is actually never called because "MFUUID" is not in the yaml file.  Also passing a string into DaosContainer.open() as the `cuuid` will cause an AttributeError (it expects a UUID object) and will only end up testing that a python str object has no hex attribute.

I'd recommend removing L134-135 instead of voiding the pylint error.